style.css

```diff
@@ -0,0 +1,76 @@
+.background-image-style {
+  width: 100%;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: 2px 4px;
+}
+
+.feature-story-image {
+  height: 170px;
+  background-image: url("assets/latest-news-5.webp");
+}
+
+.top-story-image {
+  height: 250px;
+  background-image: url("assets/latest-news-5.webp");
+}
+
+.my-turn-item a {
+  text-decoration: none;
+}
```

Comment:
Avoid using custom styling where bootstrap styling is possible. Example in the above, you can use `text-decoration-none`  bootstrap classes. Please check the rest of your styling as well.